docs/docs/strings.md

```diff
@@ -199,6 +199,15 @@ Returns true if a string contains another string.
 
 To find the index of a given substring, use the `.find()` method. This method takes an optional second parameter which can be used to skip the first `n` number of appearances of the substring. This method returns `-1` if the substring could not be found. Otherwise, it returns the index of the string.
 
+```cs
```

Comment:
I think there's a little bit of a mix up with the docs here, the find examples have went under `.findLast` and the `findLast` have went under `.find`